README.md

```diff
@@ -113,35 +113,34 @@ declared in the `form` tag.
 ### `a.cancel`
 
 Any `a` tag that has a `data-cancel-closest` attribute defined will
-trigger the cancel event handler. This simply removes from the DOM any elements
+trigger the "cancel" event handler. This simply removes from the DOM any elements
 found using the selector defined in the `data-cancel-closest` attribute:
 
 ```html
-<a href="#" data-cancel-closest=".edit-form" class="btn btn-secondary">
+<a href="#" data-cancel-closest=".edit-form" class="btn btn-secondary ajax">
```

Comment:
ajax class isn’t required and should not be used here or else it will trigger and Ajay request. Cancel is 100% client side. 

https://github.com/eldarion/eldarion-ajax/blob/master/src/eldarion-ajax-core.js#L141

Reply:
Removed. Thanks for the explanation.